discovery-provider/src/monitors/database.py

```diff
@@ -0,0 +1,49 @@
+import json
+import logging # pylint: disable=C0302
+import sqlalchemy
+
+logger = logging.getLogger(__name__)
+
+
+def get_database_liveness(**kwargs):
+    db = kwargs['db']
+    try:
+        with db.scoped_session() as session:
+            q = sqlalchemy.text(
+                'SELECT 1'
+            )
+            session.execute(q).fetchone()
+            return str(True)
+    except Exception:
+        return str(False)
+
+
+def get_database_size(**kwargs):
+    db = kwargs['db']
```

Comment:
instead of copying 
```
    db = kwargs['db']
    with db.scoped_session() as session:
```
everywhere, is there a cleaner solution?

Reply:
I couldn't think of anything better. Hopefully the comments help, but lmk if you have thoughts!